### DISCUSSION TITLE
How do I tell FSearch to always use the default application?

### DISCUSSION THREAD

**Discussion by myarbrou-rh** [cboxdoerfer/fsearch#378] [Q&A] [answered]
Currently whenever I double-click on a file in FSearch it prompts me with an "open with..." dialog. How do I make it skip this question and just open with the default app?

md and txt files currently set to open with text editor
![Screenshot from 2022-08-09 09-48-26](https://user-images.githubusercontent.com/89927559/183680644-8b328095-f50f-4f7b-92ad-2cc1b4019755.png)

which works in the regular file explorer, but when opening from FSearch it always asks which app to open with
![Screenshot from 2022-08-09 09-50-49](https://user-images.githubusercontent.com/89927559/183681054-2ce1fb1a-6e77-44a4-abb9-736a10706974.png)

**Comment by cboxdoerfer** [comment, marked as answer]
Hi, that's a limitation of the Flatpak and Snap versions — I suppose you're using one of those (installed from your software center). Both prevent FSearch from opening or knowing about you're installed applications for security reasons. So when FSearch attempts to open a file the system takes over and opens an application chooser, where you have to select the application manually. I know this is annoying, but I'm not sure if there's anything I can do to improve that.

One thing to note is that once you've picked the same application three times in the dialog, it remembers that choice and won't prompt you again. So it gets less frustrating over time.

But of course the best way to avoid this, is to use the "native" FSearch packages instead, which are not limited by such things. What distribution are you using?

**Reply by myarbrou-rh** [reply]
currently on RHEL 8.4. The choice being remembered would work for me, but I haven't noticed that behavior. Is that per file or per extension? And does it reset when i close fsearch?

**Reply by myarbrou-rh** [reply]
Nice! Okay installing via the instructions in your readme worked perfectly. Thank you!

**Reply by cboxdoerfer** [reply]
> currently on RHEL 8.4. The choice being remembered would work for me, but I haven't noticed that behavior. Is that per file or per extension? And does it reset when i close fsearch?

That's per file type and the choice is remembered when you close FSearch.

> Nice! Okay installing via the instructions in your readme worked perfectly. Thank you!

:+1:
